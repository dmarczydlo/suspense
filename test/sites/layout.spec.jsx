import React from 'react';
import Layout from '../../src/layout';

describe('layout', () => {
    let wrapper = null;

    it('should be rendered', () => {
        wrapper = global.shallow(
            <Layout>
                <div>
                    {'abcd'}
                </div>
            </Layout>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
