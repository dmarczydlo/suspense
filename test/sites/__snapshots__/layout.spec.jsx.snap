// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout should be rendered 1`] = `
<div>
  <header />
  <div>
    abcd
  </div>
  <footer />
</div>
`;
